data/mods/gen5/pokedex.ts

```diff
@@ -67,6 +67,10 @@ export const BattlePokedex: {[k: string]: ModdedSpeciesData} = {
 		inherit: true,
 		baseStats: {hp: 80, atk: 110, def: 130, spa: 55, spd: 65, spe: 45},
 	},
+	farfetchd: {
+		inherit: true,
+		name: "Farfetch'd",
+	},
```

Comment:
PS has never accurately simulated past-gen names, so it's somewhat weird to do it here.

Several other name changes also involve ID changes, which further complicates things.

But if we were to be consistent with non-ID name changes like `SolarBeam` -> `Solar Beam` and `Compoundeyes` -> `Compound Eyes`, we would leave this off, too.

Reply:
(Name legality changes, if we do implement them, can be handled elsewhere.)

Reply:
```suggestion
```

In hindsight, I don't think there's any sense in which the Gen 5 difference is remotely meaningful to PS.